SharpBucket/V2/EndPoints/RepositoriesEndPoint.cs

```diff
@@ -221,12 +218,12 @@ public class RepositoriesEndPoint{
 
         #region Commits Resource
 
-        internal CommitInfo ListCommits(string accountName, string repository, string branchortag = null ){
+        internal List<Commit> ListCommits(string accountName, string repository, string branchortag = null, int max = 0){
```

Comment:
Maybe we should allow to set the max in all the paginated calls?
Should the default be to get all the results? Just the first page?